pkg/arvo/app/chat-hook.hoon

```diff
@@ -110,20 +131,23 @@
     =/  chat-path  [%mailbox path.act]
     ?:  (~(has by synced) path.act)
       [~ this]
-    =.  synced  (~(put by synced) path.act our.bol)
+    =:  synced  (~(put by synced) path.act our.bol)
+        allow-history  (~(put by allow-history) path.act allow-history.act)
+    ==
     :_  (track-bone chat-path)
     %+  weld
       [ost.bol %peer chat-path [our.bol %chat-store] chat-path]~
     (create-permission [%chat path.act] security.act)
   ::
       %add-synced
     ?>  (team:title our.bol src.bol)
-    =/  chat-path  [%mailbox (scot %p ship.act) path.act]
+    =/  chat-path=path  [%mailbox (scot %p ship.act) path.act]
     ?:  (~(has by synced) [(scot %p ship.act) path.act])
       [~ this]
     =.  synced  (~(put by synced) [(scot %p ship.act) path.act] ship.act)
+    =/  history=path  ?:(ask-history.act /0 /~)
```

Comment:
Wait, do we not want the empty path `/` here for the second case? Pretty sure it's more common to do `/some/resource` and then parse the absence of a trailing `/index`, instead of doing `/some/resource/nothing` explicitly. Certainly have never seen suffix `/~` before.

Reply:
It's fine either way, but I actually like the /~ convention.